apps/rra/index.js

```diff
@@ -119,90 +119,90 @@ module.exports = {
     },
     '/higherSkill1': {
       behaviours: [higherSkillBehaviour],
-      fields: ['higherRraSkill', 'higherRraScores', 'higherRraEvidence'],
+      fields: ['higherSkill1', 'higherScores1', 'higherEvidence1'],
       next: '/higherSkill2',
       continueOnEdit: true
     },
     '/higherSkill2': {
       behaviours: [higherSkillBehaviour],
-      fields: ['higherRraSkill2', 'higherRraScores2', 'higherRraEvidence2'],
+      fields: ['higherSkill2', 'higherScores2', 'higherEvidence2'],
       next: '/higherSkill3',
       continueOnEdit: true
     },
     '/higherSkill3': {
       behaviours: [higherSkillBehaviour],
-      fields: ['higherRraSkill3', 'higherRraScores3', 'higherRraEvidence3'],
+      fields: ['higherSkill3', 'higherScores3', 'higherEvidence3'],
       next: '/higherSkill4',
       continueOnEdit: true
     },
     '/higherSkill4': {
       behaviours: [higherSkillBehaviour],
-      fields: ['higherRraSkill4', 'higherRraScores4', 'higherRraEvidence4'],
+      fields: ['higherSkill4', 'higherScores4', 'higherEvidence4'],
       next: '/higherSkill5',
       continueOnEdit: true
     },
     '/higherSkill5': {
       behaviours: [higherSkillBehaviour],
-      fields: ['higherRraSkill5', 'higherRraScores5', 'higherRraEvidence5'],
+      fields: ['higherSkill5', 'higherScores5', 'higherEvidence5'],
       next: '/higherSkill6',
       continueOnEdit: true
     },
     '/higherSkill6': {
       behaviours: [higherSkillBehaviour],
-      fields: ['higherRraSkill6', 'higherRraScores6', 'higherRraEvidence6'],
+      fields: ['higherSkill6', 'higherScores6', 'higherEvidence6'],
       next: '/cpd'
     },
     '/cpd': {
       fields: ['cpdDescription'],
       continueOnEdit: true,
-      next: '/rraSupportingDocumentsUpload'
+      next: '/supportingDocumentsUpload'
     },
-    '/rraSupportingDocumentsUpload': {
+    '/supportingDocumentsUpload': {
```

Comment:
the view for this page needs to be renamed as it is still rraSupportingDocumentsUpload.html and files are no longer saving